test/test_kafka_streams_examples.py

```diff
@@ -49,7 +49,9 @@ def test_default_config(self):
         #       we should also validate some REST API responses (see `test_kafka_rest.py`)?
 
     def test_secure_config(self):
-        output = self.cluster.run_command_on_service("secure-config", "bash -c 'while [ ! -f /tmp/config-is-done ]; do echo waiting && sleep 1; done; echo PASS'")
+        output = self.cluster.run_command_on_service(
+            "secure-config",
```

Comment:
Can also add the line break as self.cluster.run_command_on_service("secure-config",

Reply:
The 2nd line is still a bit long / either way is good style if it works.